docker/Dockerfile

```diff
@@ -14,6 +14,7 @@ ARG TAG_NAME=unknown
 ADD . /graph-node
 
 RUN cd /graph-node \
+    && apt-get update && apt-get install -y cmake \
```

Comment:
Can we remove this [line's](https://github.com/graphprotocol/graph-node/blob/28803bf48a9ee8df29342445360d87699a23f9e3/docker/Dockerfile#L77) `update`?

Reply:
No, they happen in two unrelated containers